tools/actions/merge.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/sh
+
+# Requires [gh](https://cli.github.com/), [jq](https://jqlang.github.io), git, and grep. Also awk if you pass a URL.
+
+# This script can be used to "purple-merge" PRs that are supposed to land as a single commit, using the "Squash and Merge" feature of GitHub.
+# To land a PR with this tool:
+# 1. Run `git node land <pr-id-or-url> --fixupAll`
+# 2. Copy the hash of the commit at the top of the PR branch.
+# 3. Run `tools/actions/merge.sh <pr-id-or-url> <commit-hash>`.
+
+set -xe
+
+pr=$1
+commit_head=$2
+shift 2 || (echo "Expected two arguments" && exit 1)
```

Comment:
Is it just syntactic preference, or is there another reason to use one form or the other?

Reply:
`{}` instead of `()` just doesn't spawn subshell (placebo performance difference in this case)
`;` instead of `&&` is to exit immediately even if `echo` fails (for example: output was redirected to logfile and something goes wrong, like out of space or no permission)

`set -e` nullifies the not-exiting risk (and swallows exit code if it was something custom), but it never hurts to provide better examples for folks who might copy-paste this.